apps/paper/pages/programmable-intents/operators.mdx

```diff
@@ -1 +1,22 @@
 # Operators
+
+The Light Layer (LL) is a dynamic blockchain infrastructure that is maintained and secured by two distinct types of operators: Validators and Auctioneers. These operators are crucial for the robust operation of the LL, ensuring not only the security of the network but also the efficiency and reliability of cross-chain transactions on behalf of users. Such transactions necessitate explicit authorization from users, which is typically obtained through specific signature permissions at the initiation phase of cross-chain transactions. The operational architecture of the LL is delineated as follows:
+
+- A total of \(k\) blockchain chains,
+- \(N\) validators,
+- \(W\) auctioneers,
+- A constant factor, \(c = 3\), serving as the default parameter but adjustable based on the network's evolving requirements for security and the management of capital flows.
+
+## Validators
+
+Validators are instrumental in upholding the integrity and security of the LL. Their responsibilities extend beyond mere transaction validation; they are tasked with maintaining a synchronized global state across all Ethereum Virtual Machines (EVMs), thereby ensuring the seamless interoperability and functionality of the network at all times. In fulfilling these duties, validators are subject to a set of stringent criteria within the Light protocol, namely:
```

Comment:
> maintaining a synchronized global state across all Ethereum Virtual Machines (EVMs), thereby ensuring the seamless interoperability and functionality of the network at all times

This would probably have to be an "optimistic" view that is challenged/settled w/ a bond through some sort of game theory based approach, as running a full node on the side does not guarantee a consensus of a particular chain A, much less multiple chains

Reply:
Or on the other hand can be a 8/11 multi sig where node operators verify together to come to consensus about a state of chain A